letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -119,6 +119,15 @@ elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-azure" ]; then
     fi
     PROVIDER_ARGUMENTS+=("--authenticator" "${DNS_PROVIDER}" "--${DNS_PROVIDER}-config" "/data/azure_creds")
 
+# INWX
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-inwx" ]; then
+    bashio::config.require 'dns.inwx_url'
+    bashio::config.require 'dns.inwx_username'
+    bashio::config.require 'dns.inwx_password'
+    bashio::config.require 'dns.inwx_shared_secret'
+    echo -e "dns_inwx_url           = $(bashio::config 'dns.inwx_url')\ndns_inwx_username      = $(bashio::config 'dns.inwx_username')\ndns_inwx_password      = $(bashio::config 'dns.inwx_password')\ndns_inwx_shared_secret = $(bashio::config 'dns.inwx_shared_secret')\n" > /data/dnsapikey
```

Comment:
Please follow the established pattern and do this in [cont-init.d/file-structure.sh](https://github.com/home-assistant/addons/blob/master/letsencrypt/rootfs/etc/cont-init.d/file-structure.sh)

Reply:
Changed to the established pattern.